xilem/src/view/prop.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright 2025 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::cmp::PartialEq;
+use std::marker::PhantomData;
+
+use masonry::core::Property;
+use xilem_core::{MessageContext, Mut};
+
+use crate::core::{View, ViewMarker};
+use crate::{Pod, ViewCtx, WidgetView};
+
+/// A view that adds a property `P` or overrides a previously defined property `P`.
```

Comment:
We maybe should point to the `Style` trait here? And we definitely should point to however you construct this (as a non-Xilem crate)

Reply:
As it is now?